lib/acme/client.rb

```diff
@@ -151,13 +151,13 @@ def deactivate_authorization(url:)
   end
 
   def challenge(url:)
-    response = get(url)
+    response = post_as_get(url)
     arguments = attributes_from_challenge_response(response)
     Acme::Client::Resources::Challenges.new(self, **arguments)
   end
 
   def request_challenge_validation(url:, key_authorization:)
-    response = post(url, payload: { keyAuthorization: key_authorization })
+    response = post(url, payload: {})
```

Comment:
I don't understand why the key_authorization is being removed?

Reply:
It’s mentioned in this issue: 

https://github.com/letsencrypt/pebble/issues/192

Pebble will raise an error if key authorization is in the request payload:

https://github.com/letsencrypt/pebble/blob/master/wfe/wfe.go#L2135

Reply:
Thanks for the clarification